label_variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "tags" {
+  description = "[Required unless var.context used] Default tags."
```

Comment:
tags are not required, they are optional in many cases
can you update the description?

Reply:
done